static/usage/v7/content/theming/safe-area/index.md

```diff
@@ -0,0 +1,33 @@
+import Playground from '@site/src/components/global/Playground';
+
+import javascript from './javascript.md';
+
+import react_main_tsx from './react/main_tsx.md';
+import react_main_css from './react/main_css.md';
+
+import vue from './vue.md';
+
+import angular_example_component_html from './angular/example_component_html.md';
+import angular_example_component_css from './angular/example_component_css.md';
+
+<Playground
+  version="7"
+  code={{
+    javascript,
+    react: {
+      files: {
+        'src/main.tsx': react_main_tsx,
+        'src/main.css': react_main_css,
+      },
+    },
+    vue,
+    angular: {
+      files: {
+        'src/app/example.component.html': angular_example_component_html,
+        'src/app/example.component.css': angular_example_component_css,
+      },
+    },
+  }}
+  src="usage/v7/content/theming/safe-area/demo.html"
+  devicePreview
```

Comment:
I think this needs `includeIonContent={false}` so you don't end up with two ion-contents

Reply:
Where should I be looking for the two ion-contents? I'm not seeing them.

Reply:
Nvm, I found it! I've learned something new about the Playground. Thank you!